### PR TITLE
add a api named scrape for different mysql instance

### PR DESCRIPTION
**Pull request by soul12345**
Signed-off-by: soul12345 <735205371@qq.com>

### CODE REVIEW

File: mysqld_exporter.go [hunk 1]
```diff
@@ -160,6 +160,16 @@ func customizeTLS(sslCA string, sslCert string, sslKey string) error {
 	return nil
 }
 
+func getdsnByRequest(r *http.Request)string{
+	user := r.URL.Query().Get("user")
+	password := r.URL.Query().Get("pass")
```
Comment (SuperQ) [inline mysqld_exporter.go]: Passing credentials over URL params is insecure. This is not allowed.

Comment (soul12345) [reply mysqld_exporter.go]: thx for reply. How about  write credentials in config , url will carry a parameter key  for point credentials

Comment (SuperQ) [reply mysqld_exporter.go]: Yes, that's more reasonable. This is how we do things with the snmp_exporter.

**Comment by SuperQ** [comment]
There's no need to open and close PRs. Simply push new commits to the existing PR to make changes.

**Pull request closed without merging**